helpers/purchase.asset.helper.js

```diff
@@ -0,0 +1,15 @@
+const assetPurchaseConfig = function assetConfigCalc(investment, portfolioConfig) {
+  const purchaseConfig = {
+    pBTC: Math.round(investment * portfolioConfig.btcWeight),
+    pETH: Math.round(investment * portfolioConfig.ethWeight),
+    pUSDC: Math.round(investment * portfolioConfig.usdcWeight),
+  };
+  purchaseConfig.pBTC += investment - (
```

Comment:
What does this do? (the +=)

Reply:
Okay great, thanks for the explanation.(BTW Buda works with decimals for CLP, but I like your solution)